core/src/main/scala/org/genivi/sota/core/data/InstallHistory.scala

```diff
@@ -15,17 +15,22 @@ import java.util.UUID
  * @param updateId Id of update this operation belongs to.
  * @param resultCode The status of operation.
  * @param resultText The description of operation.
+ * @param vin The vin of the vehicle the operation was performed on
+ * @param namespace The namespace for the given row
  */
 case class OperationResult(
-  id : String,
-  updateId: UUID,
-  resultCode : Int,
-  resultText : String
+  id        : String,
+  updateId  : UUID,
+  resultCode: Int,
+  resultText: String,
+  vin       : Vehicle.Vin,
+  namespace : Namespace
```

Comment:
The client will not know what is the namespace, so it will not be able to send any, this should come from the authentication headers, which come from somewhere else...